### ISSUE TITLE
Since v0.40.0, touch event `MoveUpdate` cannot be called continuously after tap

### ISSUE THREAD

**Issue by HTsuruo** [imaNNeoFighT/fl_chart#1127]
Hi, thanks to your awesome chart package.

Since v0.40.0, touch event `MoveUpdate` cannot be called continuously after tap(= required longPress).
Difference in behavior is here.

| v0.36.4 | v0.40.0~ |
| --- | --- |
| ![need_not_to_longpress](https://user-images.githubusercontent.com/12729025/186108735-4059661f-00b4-48f6-b65b-da86cfb7de7d.gif) | ![required_longpress](https://user-images.githubusercontent.com/12729025/186108621-6a579651-217e-45a0-b3d3-2b3daedf4444.gif) |

Is the behavior expected?

I think, it's change is not good for users.
Some users cannot aware that the chart enable for them to trace line and show tooltip because of need to wait for certain amount of time.
Therefore, I would like to revert to v0.36.0 behavior.

## To Reproduce
Just use `line_chart_sample_1.dart`
https://github.com/imaNNeoFighT/fl_chart/blob/master/example/lib/line_chart/samples/line_chart_sample1.dart

Thanks.

**Comment by Divinoart** [comment]
I suggest we have a flag for this, to set the long press to be true or false.
user are not mostly aware that they need to longPress

**Comment by Divinoart** [comment]
Found a fix for it,
Not the best practice but you'll need to edit the package.

Goto lib/src/chart/base/base_chart/render_base_chart.dart

then update line 71 to this:

![Screenshot 2022-08-23 at 9 44 21 PM](https://user-images.githubusercontent.com/60929201/186324073-1f2782a6-b273-41da-b0c7-5d4e4c8fe386.png)


`_longPressGestureRecognizer = LongPressGestureRecognizer(duration: const Duration(milliseconds: 0));`

The trick is, it sets the LongPressGestureRecognizer.duration to be 0 milliseconds instead of 500 from kLongPressTimeout.


Suggestion to the dev team is to make this a feature flag so people can enable or disable LongPress feature

**Comment by HTsuruo** [comment]
@Divinoart 

Hi, thanks for your response.

> I suggest we have a flag for this, to set the long press to be true or false.

Your approach is better, so I agree with you.

**Comment by HTsuruo** [comment]
@Divinoart 

>_longPressGestureRecognizer = LongPressGestureRecognizer(duration: const Duration(milliseconds: 0));
> The trick is, it sets the LongPressGestureRecognizer.duration to be 0 milliseconds instead of 500 from kLongPressTimeout.

Good Job👍
Your solution works expected behavior.

```dart
_longPressGestureRecognizer = LongPressGestureRecognizer(duration: Duration.zero);
```

| Duration.zero（v0.55.1） |
| --- |
| ![duration zero](https://user-images.githubusercontent.com/12729025/186339913-64bdd3af-198c-429e-9a72-910041343a61.gif) |

**Comment by Divinoart** [comment]
> @Divinoart
> 
> > _longPressGestureRecognizer = LongPressGestureRecognizer(duration: const Duration(milliseconds: 0));
> > The trick is, it sets the LongPressGestureRecognizer.duration to be 0 milliseconds instead of 500 from kLongPressTimeout.
> 
> Good Job👍 Your solution works expected behavior.
> 
> ```dart
> _longPressGestureRecognizer = LongPressGestureRecognizer(duration: Duration.zero);
> ```
> 
> Duration.zero（v0.55.1）
> ![duration zero](https://user-images.githubusercontent.com/12729025/186339913-64bdd3af-198c-429e-9a72-910041343a61.gif)

Great!

**Comment by HTsuruo** [comment]
Added `longPressGestureDuration` property. 
https://github.com/HTsuruo/fl_chart/tree/feature/tsuruo/enable-customize-longpress-duration

I hesitated that optional property should take a boolean true/false, but decided that `Duration` property because it's better for flexibility.

**Comment by HTsuruo** [comment]
Resolved by #1128

**Comment by imaNNeoFighT** [comment]
> Resolved by #1128

Thank you for contributing.
It will be available in the next version soon.

**Comment by imaNNeo** [comment]
[0.60.0](https://github.com/imaNNeo/fl_chart/releases/tag/0.60.0) is just released. Check it out!
